db/seeds.rb

```diff
@@ -1,8 +1,455 @@
+                            starts_on: Date.new(2022, 1, 1),
+                            ends_on: Date.new(2022, 12, 31)
+                          },
+                          {
+                            year: 2022,
+                            name: '7º Ano',
+                            starts_on: Date.new(2022, 1, 1),
+                            ends_on: Date.new(2022, 12, 31)
+                          },
+                          {
+                            year: 2022,
+                            name: '8º Ano',
+                            starts_on: Date.new(2022, 1, 1),
+                            ends_on: Date.new(2022, 12, 31)
+                          },
+                          {
+                            year: 2022,
+                            name: '9º Ano',
+                            starts_on: Date.new(2022, 1, 1),
+                            ends_on: Date.new(2022, 12, 31)
+                          },
+                          {
+                            year: 2023,
+                            name: '5º Ano',
+                            starts_on: Date.new(2023, 1, 1),
+                            ends_on: Date.new(2023, 12, 31)
+                          },
+                          {
+                            year: 2023,
+                            name: '6º Ano',
+                            starts_on: Date.new(2023, 1, 1),
+                            ends_on: Date.new(2023, 12, 31)
+                          },
+                          {
+                            year: 2023,
+                            name: '7º Ano',
+                            starts_on: Date.new(2023, 1, 1),
+                            ends_on: Date.new(2023, 12, 31)
+                          },
+                          {
+                            year: 2023,
+                            name: '8º Ano',
+                            starts_on: Date.new(2023, 1, 1),
+                            ends_on: Date.new(2023, 12, 31)
+                          },
+                          {
+                            year: 2023,
+                            name: '9º Ano',
+                            starts_on: Date.new(2023, 1, 1),
+                            ends_on: Date.new(2023, 12, 31)
+                          }])
+
+GRADES = ['5º Ano', '6º Ano', '7º Ano', '8º Ano', '9º Ano'].freeze
+
+enrollments = []
+
+# PROGRESSION equals to a random number (0 or 1) that defines if a student's going to have
+# more than one course enrolled, in a progression of grades e.g 5º Ano (2013), 6º Ano (2016) and so forth.
+# if there's a progression then it creates the necessary enrollments, if it reaches 2023 or the the last GRADES
+# index + 1, then it doesn't iterate
+students.each do |student|
+  PROGRESSION = rand(0..1).to_i
+  grade = GRADES[rand(0..4)]
+  random_year = rand(2012..2023).to_i
+
+  if PROGRESSION.zero? || random_year == 2023
+    course = courses.select { |c| c.year == random_year && c.name == grade }
```

Comment:
I understood the meaning of `c` but It's not as clear as it could be. If you're iterating through an array of courses, name it "course".